README.md

```diff
@@ -1,4 +1,4 @@
-# http4s giter8 template
+# http4s giter8 template [![Build Status](https://travis-ci.org/http4s/http4s.g8.svg?branch=master)](https://travis-ci.org/http4s/http4s.g8)
```

Comment:
Will this work since the master branch is gone, replaced by 0.17?

Without the query param, will it use the GitHub default branch?  That would be ideal.

Reply:
I believe so, as this is what it selected when I selected for branch 0.17